lib/config/presets/internal/replacements.ts

```diff
@@ -901,6 +902,18 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'standard-version-to-commit-and-tag': {
+    description:
+      '`standard-version` is now maintained as `commit-and-tag-version`.',
+    packageRules: [
+      {
+        matchDatasources: ['npm'],
```

Comment:
```suggestion
        matchCurrentVersion: '^9.0.0',
        matchDatasources: ['npm'],
```

Only upgrade users from v9, if somebody uses an older version, then we shouldn't suggest (force) a replacment